forc-plugins/forc-crypto/src/args.rs

```diff
@@ -6,8 +6,20 @@ use std::{
     path::Path,
 };
 
+forc::cli_examples! {
+    [ Hashes data passing as an argument with SHA256 => crypto "sha256" "test" ]
+    [ Hashes data passing as an argument with Keccak256 => crypto "keccak256" "test" ]
+    [ Hashes a file path as an argument with Sha256 => crypto "sha256" "src/args.rs" ]
+    [ Hashes a file path as an argument with Keccak256 => crypto "keccak256" "src/args.rs" ]
+}
+
 #[derive(Debug, Clone, clap::Args)]
-#[clap(author, version, about = "Hashes the argument or file with this hash")]
+#[clap(
+    author,
+    version,
+    about = "Hashes the argument or file with this hash",
```

Comment:
```suggestion
    about = "Hashes the argument or file with this algorithm",
```

I assume this refers to the algorithm used to create a hash and not the output hash itself